content/en/developers/metrics/_index.md

```diff
@@ -17,34 +18,40 @@ further_reading:
   text: "Official and Community created API and DogStatsD client libraries"
 ---
 
-A Datadog metric is defined by the properties below. Refer to the [Metrics Introduction documentation][1] to learn how to graph metrics within Datadog.
+## Overview
 
-| Property         | Description                                                                                                                                               |
-|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `<METRIC_NAME>`  | The [name of your metric](#naming-metrics).                                                                                                               |
-| `<METRIC_VALUE>` | The value of your metric.                                                                                                                                 |
-| `<TIMESTAMP>`     | The timestamp associated with the metric value. **Note**: Metric timestamps cannot be more than 10 minutes in the future or more than 1 hour in the past. |
-| `<CONTEXT>`      | The set of tags associated with your metric.                                                                                                              |
-| `<METRIC_TYPE>`  | The type of your metric. See the [metric type documentation][2].                                                                                          |
-| `<INTERVAL>`     | If the `<TYPE>` of the metric is [RATE][3] or [COUNT][4], it defines the corresponding [interval][5].                                                                    |
+If a metric is not submitted from one of the [350+ Datadog integrations][1] it's considered a custom metric<sup>(1)</sup>. Custom metrics help you track your application KPIs: number of visitors, average customer basket size, request latency, or performance distribution for a custom algorithm.
 
-If a metric is not submitted from one of the [350+ Datadog integrations][6], it's considered a [Custom Metric][7]. **Note**: Some standard integrations [emit custom metrics][8].
+A custom metric is identified by **a unique combination of a metric's name and tag values (including the host tag)**. In general, any metric you send using [DogStatsD][2] or through a [custom Agent Check][3] is a custom metric.
 
-### Naming metrics
+**Note**: Users with the Datadog Admin roles can see the total custom metrics per hour and the top 500 custom metrics for their account in the [usage details page][4]. Learn more about [how custom metrics are counted][5].
```

Comment:
**Note**: Users with the Datadog Admin roles can see the monthly average number of custom metrics per hour and the top 500 custom metrics for their account in the [usage details page][4]. Learn more about [how custom metrics are counted][5].